library/alloc/src/collections/binary_heap/mod.rs

```diff
@@ -1897,3 +1910,59 @@ impl<'a, T: 'a + Ord + Copy, A: Allocator> Extend<&'a T> for BinaryHeap<T, A> {
         self.reserve(additional);
     }
 }
+
+#[cfg(kani)]
+#[unstable(feature="kani", issue="none")]
+mod verify {
+    use super::*;
+
+    // TODO: Kani reports as failing property "Only a single top-level call", which does not
```

Comment:
what exactly is the error?

Reply:
Following up on this: can you extend the TODO to say what function it's complaining about for the single top-level call?

Reply:
Otherwise, this PR LGTM, but I am curious if we can do a bit of debugging on this harness before merging it as a TODO. If we really get stuck, I'll approve.